app/controllers/cohorts/clients_controller.rb

```diff
@@ -135,12 +135,13 @@ def new
       elsif @client_ids.present?
         @client_ids = @client_ids.strip.split(/\s+/).map { |m| m[/\d+/].to_i }
         @clients = client_scope.where(id: @client_ids)
-      elsif params.dig(:q, :full_text_search).present?
-        @q = client_source.ransack(params[:q])
+      elsif params.dig(:search_form, :q).present?
+        search_setup(scope: :full_text_search)
```

Comment:
Since the next line references `@search`, I'd like `search_setup` to return a value:
```suggestion
        @search = search_setup(scope: :full_text_search)
```

Reply:
This would apply each use below...

Reply:
Yeah, we could do that.  I was thinking of it kinda like the before actions where it takes care of setting the instance variable (which is does.)  So that assignment is a no-op, but maybe makes it more readable.